### PR TITLE
teams: split does not permit parent PE wrap-around

### PR DESCRIPTION
**Pull request by davidozog**
# Summary of changes
This attempts to clarify that no team split "wrap-around" is allowed.

I found it a little tricky to specify, so please feel free to propose new or adjusted text.

I'm relying on #1 to account for the `start` argument description.

Changelog entry proposed here:
https://github.com/kwaters4/specification/pull/6/files

### CODE REVIEW

**Comment by davidozog** [comment]
@wokuno @maawad - ready for review (looks like I don't have permission to request directly)

File: content/shmem_team_split_strided.tex
Comment (maawad) [inline content/shmem_team_split_strided.tex]:
```suggestion
When $stride$ is nonzero, a newly created team must only include \acp{PE} whose subsequent parent \ac{PE}
```

**Review by maawad — APPROVED** [review]
Thanks, David, for fixing this. I made a minor suggestion but I might be getting too verbose. Looks good to me either way.

**Review by wokuno — APPROVED** [review]
Looks good. Could you just merge this into the `sec/teams-ctx` branch

**Comment by davidozog** [comment]
> Looks good. Could you just merge this into the `sec/teams-ctx` branch

Good catch!  I was apparently working too quickly...  I just updated it - will merge in a moment unless I hear otherwise.

**Comment by davidozog** [comment]
Oh, just need 1 more approval to merge.

File: content/shmem_team_split_strided.tex
Comment (bobkierski) [inline content/shmem_team_split_strided.tex]: I would suggest moving this part of the change to the top (line 62).  The content between 62 and 69 would then clarify what is meant by this.

Comment (bobkierski) [reply content/shmem_team_split_strided.tex]: that would also mean that "implies such a wrap-around" would be changed to "implies a wrap-around".

Comment (davidozog) [reply content/shmem_team_split_strided.tex]: Done!  Can you please review the change to see if I captured it well?  Thanks, @bobkierski

**Review by bobkierski — APPROVED** [review]
Se ve bueno para mi

**Comment by davidozog** [comment]
@wokuno @maawad @bobkierski - looks like this is ready to merge, but I do not have permission to do it.  Could you please hit the big green button if you think it's ready?

**Comment by maawad** [comment]
> @wokuno @maawad @bobkierski - looks like this is ready to merge, but I do not have permission to do it. Could you please hit the big green button if you think it's ready?

Maybe @wokuno can help us. I don't have permission to merge either with this PR and #1  too.

**Pull request merged**
